docs/toc.md

```diff
@@ -0,0 +1,65 @@
+# Welcome to Multicast' documentation!
+
+## Quickstart:
+**Welcome to the Multicast Python Library! Let's get you started quickly.**
+
+**Step 1: Install Python 3**
+
+* Ensure Python 3 is installed on your system.
+
+**Step 2: Clone the Repository**
+
+* Open your terminal and run:
+  ```shell
+  git clone https://github.com/reactive-firewall/multicast.git
+  cd multicast
+  ```
+
+**Step 3: Install the Package**
+
+* Run:
+  ```shell
+  make install
+  ```
+
+**Step 4: Sending Messages**
+
+* Send a message using:
+  ```shell
+  python3 -m multicast SAY --mcast-group 224.1.1.2 --port 59595 --message "Hello, Multicast!"
+  ```
+
+**Step 5: Receiving Messages**
+
+* Receive messages by running:
+  ```shell
+  python3 -m multicast RECV --use-std --port 59595 --join-mcast-groups 224.1.1.2
+  ```
+
+**You're all set! Enjoy using Multicast for your projects.**
+
+
+
+## Contents:
```

Comment:
LGTM, but this may be because I'm unfamiliar with the guidelines:
* break down the "Markdown style guidelines (MD026)" you mentioned. Please include a few examples of both violations and adherence of the guideline. I have a feeling we're about to adopt some new project conventions 😈